language/move-native/src/rt.rs

```diff
@@ -38,7 +38,129 @@ unsafe extern "C" fn vec_cmp_eq(
     crate::vector::cmp_eq(type_ve, v1, v2)
 }
 
+#[export_name = "move_rt_str_cmp_eq"]
+unsafe fn str_cmp_eq(s1: &str, s2: &str) -> bool {
```

Comment:
I realize that this patch is a draft, but I'm adding a few minor comments while I familiarize myself with the code.

Can we replace this whole function body with `s1 == s2`?

Reply:
Initially, I had `s1 == s2`, but that compared addresses of s1 and s2, not sure why it didn’t expand to element-wise comparison.

Reply:
Does `*s1 == *s2` work?

Reply:
This seems to work. Thank you.